lib/mongooseice/udp/worker.ex

```diff
@@ -89,6 +92,14 @@ defmodule MongooseICE.UDP.Worker do
     {:noreply, next_state, timeout(next_state)}
   end
 
+  def handle_info({:udp_passive, socket},
+                  %{turn: %TURN{allocation: %TURN.Allocation{socket: socket}}} = state) do
+    n = burst_length()
+    Logger.debug(~s"Processed #{n} peer packets")
+    :inet.setopts(socket, [active: n])
```

Comment:
@erszcz, @rslota, and @arkgil: perhaps we should put a `:timer.sleep/1` here (or better arrange a `send_after/3` to a similar effect) if we're going to rate limit the socket. I'm wondering yous had something else in mind

Reply:
I suppose that what @erszcz had in mind when submitting an issue was that `active: true` may result in process' message queue growing indefinitely. I think the solution implemented by you here is sufficient, because it prevents such scenario (i.e. we'll have `n` packets in a message queue at maximum).

But maybe I don't understand something - what kind of timer did you have in mind?

Reply:
@arkgil Never mind, you're right, the solution here is sufficient. I didn't think it through properly. This is @erszcz's code BTW.